templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -22,6 +22,7 @@ limitations under the License.
 import * as gax from 'google-gax';
 import * as path from 'path';
 
+import { Transform } from 'stream';
```

Comment:
Please place this import into an `if`, we don't want to import it if there are no paging methods.

Reply:
that makes sense, thanks!